frontend/src/components/Home.js

```diff
@@ -39,6 +39,23 @@ const Home = () => {
       <h1 class="text-4xl font-bold mb-4 text-red-500">
         Welcome, {store.data.first_name} {store.data.last_name}!
       </h1>
+      <Show when={store.data.is_player}>
+        <div>
+          <Show when={!store.data.player}>
+             <h3 class="text-2xl font-bold mb-4">
+              Want to play?{" "}
+              <A href="/registration" class="font-medium text-blue-600 dark:text-blue-500 hover:underline">
+                Register!
+              </A>
+            </h3>
+          </Show>
+          <Show when={store.data.player}>
```

Comment:
This can become a `fallback` to the Show above?

Reply:
Added a fallback to the `<Show when={!store.data.player}...>`